controllers/clustersecret_controller.go

```diff
@@ -271,5 +282,9 @@ func getLabels(name string) map[string]string {
 func (r *ClusterSecretReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&v12.ClusterSecret{}).
+		WithEventFilter(predicate.Funcs{
+			UpdateFunc: func(event event.UpdateEvent) bool {
```

Comment:
do we need this for create event too ?

Reply:
I don't think so. If we filter out the create event, in case the operator goes down by some period when it gets back it might miss some new namespace created during this period of downtime.
That's why the CreateFunc is empty.

Reply:
noted